travel_platform/skyscanner_data/skyscanner_data.py

```diff
@@ -13,29 +13,32 @@
         "outbounddate": "2017-07-15", "inbounddate": "2017-07-18"
 }
 
+class SkyscannerData(object):
 
-def cheapest_quotes(params):
-    result = flights_cache_service.get_cheapest_quotes(**params).parsed
-    return result
+    def __init__(self, params):
+        self.params = params
```

Comment:
I really don't like this approach.

It's much better to these pass parameters into function instead of class instance.

Btw, your tests also fails because of this issue. If you fix this part, your tests should work with parameters as you expect.